scripts/review/diff.ts

```diff
@@ -8,7 +8,7 @@ type Options = {
   verbose: boolean;
 };
 
-type Deployment = { [key: number]: string };
+type Deployments = { [chainID: number]: string | string[] };
```

Comment:
ubernit: Typically, in JS we use camelCase with lower-case acronyms: 
```suggestion
type Deployments = { [chainId: number]: string | string[] };
```

Reply:
Done 👍🏾